pkg/kubelet/kubelet.go

```diff
@@ -2202,6 +2202,11 @@ func (kl *Kubelet) updateRuntimeUp() {
 	// If RuntimeReady is not set or is false, report an error.
 	if runtimeReady == nil || !runtimeReady.Status {
 		klog.Errorf("Container runtime not ready: %v", runtimeReady)
+		if !kl.runtimeState.isInitialized() {
```

Comment:
`initalized will be false until containerd or cri-o is running` .  This is not the case, If `Status` returns error, And `intialized` is `false`, we would set the `lastBaseRuntimeSync(not baseRuntimeSyncThreshold for reminder)` to an outdated time (which is `time.Now - baseRuntimeSyncThreshold - 1s`) .  This would cause `kubelet.runtimeErrors` report `container runtime is down` immediately and also indicate runtime has been initialized( which is expected).

Reply:
I mean what if the `Status` function above returns error directly. Check the line 2177